src/controllers/admin/evaluation-result-controller.ts

```diff
@@ -11,90 +11,31 @@ import * as EvaluationResultService from "../../services/evaluation-result-servi
  * @param req.query.evaluation_administration_id - Filter by evaluation administration id.
  * @param req.query.name - Filter by name.
  * @param req.query.status - Filter by status.
+ * @param req.query.score_ratings_id - Filter by score ratings id.
+ * @param req.query.banding - Filter by banding.
+ * @param req.query.banding - Filter by sort_by.
```

Comment:
should be `sort_by` instead of `banding`

Reply:
done ^^